doc/user_guide/eds.rst

```diff
@@ -356,14 +423,10 @@ are used by default.
    
 .. code-block:: python
 
-    >>> spec_img.set_lines(["Ni_La","Ti_Ka","Al_Ka"])
-    >>> spec_img.get_lines_intensity()
-    [<Image, title: Intensity of Al_Ka at 1.49 keV from Spectrum image,
-     dimensions: (|128, 95)>,
-    <Image, title: Intensity of Ni_La at 0.85 keV from Spectrum image,
-     dimensions: (|128, 95)>,
-    <Image, title: Intensity of Ti_Ka at 4.51 keV from Spectrum image,
-     dimensions: (|128, 95)>]
+    >>> s.set_lines(['Fe_Ka', 'Pt_La'])
+    >>> s.get_lines_intensity()
+    [<Image, title: X-ray line intensity of Core shell: Fe_Ka at 6.40 keV, dimensions: (|64, 64)>,
+    <Image, title: X-ray line intensity of Core shell: Pt_La at 9.44 keV, dimensions: (|64, 64)>]
```

Comment:
Again, if we are trying to always supply "complete" examples in every snippet, there should be `s` definition here